test/user.js

```diff
@@ -0,0 +1,95 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  should = chai.should();
+
+describe('users', () => {
+  describe('/users/ POST', () => {
+    it('should create a new user without problems', () => {
+      return chai
+        .request(server)
+        .post('/users/')
+        .send({
+          firstName: 'pepito',
+          lastName: 'perez',
+          email: 'pepito.perez@wolox.com',
+          password: 'Holahola23'
+        })
+        .then(res => {
+          res.should.have.status(201);
+          dictum.chai(res);
+        });
+    });
+
+    it('should fail, email already in use for another user', () => {
```

Comment:
This is wrong, database is cleaned between each test.

Reply:
but in the first test a user is not created?

Reply:
Yes in first test the user is created, after hook delete that user (and all data in database)